electron-firmware/src/uavcan/node.h

```diff
@@ -5,6 +5,14 @@
 extern "C" {
 #endif
 
+enum {
+    INIT,
+    READY,
+    RUNNING,
+    ARRIVED
+};
```

Comment:
Could be a `enum class ElectronState {...}` for type safety, then you'll need to use `ElectronState::INIT`

Reply:
I want it to be usable from `main.c` and I'm too lazy to make it a `main.cpp`.

Reply:
fair enough